flink-connectors/flink-connector-hive/src/test/java/org/apache/flink/connectors/hive/HiveDialectITCase.java

```diff
@@ -665,6 +667,62 @@ public void testTemporaryFunctionUDAF() throws Exception {
         tableEnv.executeSql("drop temporary function if exists foo");
     }
 
+    @Test
+    public void testTemporaryFunctionUDTFInitializeWithObjectInspector() throws Exception {
```

Comment:
@xiangqiao123 Sorry to go back to it again. 
But Could you please merge the test code/logic in `testTemporaryFunctionUDTFInitializeWithObjectInspector` and `testTemporaryFunctionUDTFInitializeWithStructObjectInspector` to one method, so that the code base for the test won't grow rapidly.